docs/installation.md

```diff
@@ -17,7 +17,7 @@ wego version
 You should see:
 
 ```console
-Current Version: 0.2.2
+Current Version: 0.2.4
 GitCommit: 466001a7fec68e05c7d4474536aadc91af0dee20
```

Comment:
We should re-run the wego command and update the `GitCommit` and `BuildTime` entries here...

Reply:
:+1: Yeah that was why this was marked as draft for now